packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -91,13 +91,12 @@ import { Chart, ChartBar, ChartGroup, ChartThemeColor, ChartVoronoiContainer } f
 </div>
 ```
 
-## Multi-color (ordered), horizontal bar chart with bottom-left aligned legend
-This demonstrates zoom for both the x and y axis
-```js
+```js title=Multi-color-(ordered),-horizontal-bar-chart-with-bottom-left-aligned-legend
```

Comment:
Words like "Multi-color", "bottom-left", etc. are used thoughout the chart examples. They had used dashes already, but now appear to use white space?

Reply:
Yep, good catch. Fixing now.